types/chrome/index.d.ts

```diff
@@ -6910,7 +6913,7 @@ declare namespace chrome.runtime {
          * The lifecycle the document that opened the connection is in at the time the port was created. Note that the lifecycle state of the document may have changed since port creation.
          * @since Chrome 106.
          */
-        documentLifecycle?: string | undefined;
+        documentLifecycle?: DocumentLifecycle;
```

Comment:
For people with `exactOptionalPropertyTypes`, this matters

```suggestion
        documentLifecycle?: DocumentLifecycle | undefined;
```